src/mds/MDCache.cc

```diff
@@ -7818,11 +7818,15 @@ bool MDCache::shutdown_pass()
   }
   assert(subtrees.empty());
 
-  if (myin)
+  if (myin) {
     remove_inode(myin);
+    myin = nullptr;
```

Comment:
this is unnecessary . remove_inode does that job.

Reply:
thanks